thirdweb/README.md

```diff
@@ -0,0 +1,28 @@
+## Getting Started
```

Comment:
Please also update the top level README pointing to this directory.

Reply:
Hey brother, I'm Kristian, Developer Advocate from thirdweb I'll be taking over this PR. May I know what you mean by this revision?

Reply:
Hi, you need to update the main README as well.
This one https://github.com/ovrclk/awesome-akash/blob/master/README.md